.betterer.results

```diff
@@ -2218,6 +2218,9 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "0"],
       [0, 0, 0, "Do not use any type assertions.", "1"]
     ],
+    "public/app/features/correlations/Forms/TransformationsEditor.tsx:5381": [
+      [0, 0, 0, "Do not use any type assertions.", "0"]
```

Comment:
We need to iterate through an enum and `SupportedTransformationType[key as keyof typeof SupportedTransformationType]` seems to be how you need to do it